src/components/OperatorArea/index.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Box,
+  Flex,
+  Link,
+  ListItem,
+  Text,
+  UnorderedList,
+} from "@chakra-ui/react";
+import type { Assembly } from "jsii-reflect";
+import { ReactNode, useMemo } from "react";
+import type { Metadata } from "../../api/package/metadata";
+import { DependencyDropdown } from "../DependencyDropdown";
+
+export interface OperatorAreaProps {
+  assembly?: Assembly;
+  metadata: Metadata;
+}
+
+export function OperatorArea({ assembly, metadata }: OperatorAreaProps) {
+  const details: ReactNode[] = useMemo(() => {
+    if (!metadata) return [];
+
+    const {
+      date,
+      publisher: { username },
+      links: { repository },
+    } = metadata;
+
+    const items = [];
+
+    if (date) {
+      items.push(`Published ${date}`);
```

Comment:
Formatting for this date will be in a follow-up that will bring in [date-fns](https://date-fns.org/) to handle date formatting in our application